lib/config/error-codes/api.json

```diff
@@ -1,6 +1,16 @@
 {
   "code": 1,
   "subdomains": {
+    "base": {
+      "code": 1,
+      "errors": {
+        "invalid_value_type": {
```

Comment:
I expect a generic error with that code, not one regarding only booleans. Change either the code or the message.